images/dynamodb_local/src/image.rs

```diff
@@ -0,0 +1,85 @@
+use std::{env::var, thread::sleep, time::Duration};
+use tc_core::{Image, Container, Docker, WaitForMessage};
+
+const ADDITIONAL_SLEEP_PERIOD: &'static str = "DYNAMODB_ADDITIONAL_SLEEP_PERIOD";
+const DEFAULT_WAIT: u64 = 2000;
+const CONTAINER_IDENTIFIER: &'static str = "amazon/dynamodb-local";
+const DEFAULT_TAG: &'static str = "latest";
```

Comment:
Using latest can lead to problems with builds suddenly breaking. Bear in mind that the library is used for testing. You don't want your tests to break because of the test-setup :)

Reply:
My thoughts were that latest was a reasonable default. Callers could use the DynamoDB::with_tag_args function to pin an exact build, though probably not necessary. The use case with DynamoDB is interesting as DynamoDB local's latest will match what is running in AWS. DynamoDB Local provided by Amazon purely for testing.

Reply:
Interesting, I guess in this case `latest` is fine.